hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -495,11 +500,11 @@ def _validate_timeout(n):
 )
 
 
-def _validate_database(db, __from_db_file=False):
```

Comment:
It doesn't matter, but what prompted this name change?

Reply:
Mypy apparently can't see `__`-prefixed argument names :roll_eyes: